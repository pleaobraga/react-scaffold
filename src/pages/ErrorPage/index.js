import './ErrorPage.scss'

export { default } from './ErrorPage'
