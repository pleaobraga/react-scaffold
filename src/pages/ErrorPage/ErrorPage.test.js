import React from 'react'
import { shallow } from 'enzyme'
import ErrorPage from './ErrorPage'

describe('welcomePage', () => {
  const errorPage = shallow(<ErrorPage />)

  it('render proprely', () => {
    expect(errorPage).toMatchSnapshot()
  })
})
