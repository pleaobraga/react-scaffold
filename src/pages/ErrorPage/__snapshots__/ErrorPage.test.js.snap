// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorPage render proprely 1`] = `ShallowWrapper {}`;
